library-front/src/components/BookDetails/BookDetails.tsx

```diff
@@ -19,18 +25,25 @@ const BookDetails: FC<BookDetailsProps> = () => {
   const dialog = useRef<HTMLDialogElement | null>(null)
   const queryClient = useQueryClient()
   const navigate = useNavigate()
-  const { data } = useQuery(['books', id], async () => {
+  const [activeDialog, setActiveDialog] = useState<'delete' | 'rent' | 'return'>('delete')
```

Comment:
isDialogActive

Reply:
Da napravim vise flegova za svaki dijalog? Posto mi activeDialog govori koji je trenutno izabrani dijalog, odnosno koje dugme za otvaranje dijaloga je kliknuto. Nemam fleg koji prikazuje dijalog, to radim preko dialog ref-a